packages/rs-dpp/src/data_contract/state_transition/data_contract_update_transition/validation/basic/validate_data_contract_update_transition_basic.rs

```diff
@@ -178,11 +185,15 @@ where
                     operation.to_owned(),
                     property_name.to_owned(),
                     existing_data_contract_object
-                        .get(property_name.split_at(1).1)?
+                        .get(property_name.split_at(1).1)
+                        .ok()
```

Comment:
I'm wondering why there was an error when we're trying to get a field befoe. Now it will just supress the error with the `ok`, and that's it